### PR TITLE
Fix grammar issues on levelPage

### PR DESCRIPTION
**Pull request by CrackThrough**
No codebase changes, only strings have changed.

### CODE REVIEW

File: src/routes/LevelPage.jsx [hunk 1]
```diff
@@ -89,16 +89,16 @@ const LevelPage = ({ history }) => {
 
   const NSFWWarningSwal = () => {
     ReactSwal.fire({
-      title: 'Include NSFW Content',
+      title: 'NSFW Content Included',
       html: (
         <>
-          This level include{' '}
-          <strong>NSFW(sensational or bloody) Content</strong>.
+          This level includes {' '}
```
Comment (ppapman1) [inline src/routes/LevelPage.jsx:95]: `includes {' '}` = `includes  `(two spaces)
please replace it with `includes{' '}`

**Review by ppapman1 — CHANGES_REQUESTED** [review]
i need change

File: src/routes/LevelPage.jsx (hunk 1 above)
Comment (CrackThrough) [reply src/routes/LevelPage.jsx]: fixed in commit https://github.com/ADOFAI-gg/ADOFAI-gg-Web/pull/30/commits/53c46d2a7482c01e88e19bd1689bb62e59690c5f

**Comment by ppapman1** [comment]
lgtm

**Pull request merged**